Packs/SymantecEDR/Integrations/SymantecEDR/SymantecEDR.py

```diff
@@ -0,0 +1,2734 @@
+        """
+        return self.http_request(
+            method='POST',
+            endpoint=f'/atpapi/v2/commands/{command_id}',
+            params={},
+            json_data=payload).json()
+
+    def get_sandbox_verdict_for_file(self, sha2: str) -> dict[str, Any]:
+        """
+        Client method for get sandbox verdict for specific file sha256
+        Args:
+            sha2 (str): File SHA256
+        Returns:
+            return response json
+        """
+        return self.http_request(
+            method='GET',
+            endpoint=f'/atpapi/v2/sandbox/results/{sha2}/verdict',
+            params={},
+            json_data={}).json()
+
+    def get_file_entity(self, sha2: str) -> dict[str, Any]:
+        """
+        Client method for Get File Entity for specific SHA2
+        Args:
+            sha2 (str): File SHA256
+        Returns:
+            return response json
+        """
+        return self.http_request(
+            method='GET',
+            endpoint=f'/atpapi/v2/entities/files/{sha2}',
+            params={},
+            json_data={}).json()
+
+    def get_sandbox_status(self, command_id: str) -> dict[str, Any]:
+        """
+        Client method to Get Sanbox Status for specific file
+        Args:
+            command_id (str): sandbox command ID
+        Returns:
+            return response json
+        """
+        return self.http_request(
+            method='GET',
+            endpoint=f'/atpapi/v2/sandbox/commands/{command_id}',
+            params={},
+            json_data={}).json()
+
+    def submit_file_to_sandbox_analyze(self, payload: dict) -> dict[str, Any]:
+        """
+        Client method for sandbox analyze
+        Args:
+            payload: request body
+
+        Returns:
+            return response json
+        """
+        return self.http_request(
+            method='POST',
+            endpoint='/atpapi/v2/sandbox/commands',
+            params={},
+            json_data=payload
+        ).json()
+
+
+''' HELPER FUNCTIONS '''
+
+
+def convert_to_iso8601(timestamp: str) -> str:
+    """ Convert timestamp from iso 8601 format
+
+    Args:
+        timestamp: Any valid timestamp or provide timedelta e.g. now, "<n> days", "<n> weeks",
+        "<n> months", "1 months ago"
+
+    Returns: return timestamp in iso 8601 format.
+
+    """
+    if datetime_from_timestamp := dateparser.parse(timestamp, settings={'TIMEZONE': 'UTC'}):
+        return f'{datetime_from_timestamp.strftime(ISO8601_DATE_FORMAT)[:23]}Z'
+    else:
+        raise ValueError(f'{timestamp} could not be parsed')
+
+
+def extract_headers_for_readable_output(summary_data: list[dict]) -> list:
+    """
+    Symantec EDR formatting Readable output Header
+    Args:
+        summary_data (list[dict]): Human readable output summary data
+
+    Returns:
+        Return string headers to camel case.
+
+    """
+    if not summary_data:
+        raise DemistoException('No Readable output data found to display.')
```

Comment:
when does this happen? Do we consider it an error?

Reply:
I think this exception is unwanted, already handle as part of "page_result" if not found then it will simply show the message that no readable data to present.